sdk/highlight-next/src/next-client.tsx

```diff
@@ -22,10 +22,15 @@ export function HighlightInit({
 				(hostname) => !window.location.hostname.includes(hostname),
 			)
 
+		const optionsWithBackendUrl = {
+			...highlightOptions,
+			backendUrl: '/highlight-events',
```

Comment:
great, but we want this to only happen if we can detect that `withHighlightConfig(...)` is set up. perhaps the `withHighlightConfig` wrapper can emit a `process.env` NEXT_PUBLIC value that is picked up here?

Reply:
Can't we directly do it in `withHighlightConfig(...)` itself